### ISSUE TITLE
No alt text for fig2, fig3

### ISSUE THREAD

**Issue by Raj-R1** [Stat184-Spring2026/Sec1_CP_Blake_Raj#20]
There is no alt text defined for figure 2, and 3. It should be defined in the YAML header for every chunk. 

Suggested fix: add in alt text generated by the accessibility tool in the YAML header.
